tests/Doctrine/Tests/DbalPerformanceTestCase.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Doctrine\Tests;
+
+/**
+ * Base class for all DBAL performance tests.
+ * 
+ * Tests implemented in this class must call startTiming at the beginning
+ * and stopTiming at the end of all tests. Tests that do not start or stop
+ * timing will fail.
+ *
+ * @package Doctrine\Tests\DBAL
+ * @author Bill Schaller
+ */
+class DbalPerformanceTestCase extends DbalFunctionalTestCase
+{
+    /**
+     * time the test started
+     *
+     * @var float
+     */
+    private $startTime;
+
+    /**
+     * elapsed run time of the last test
+     *
+     * @var float
+     */
+    private $lastRunTime;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        // Reset timing vars
+        $this->startTime = $this->lastRunTime = null;
```

Comment:
Those vars will be reset on each test case anyways. The whole `setUp()` can be omitted IMO